### DISCUSSION TITLE
关于登录用户cookie及用户ip的安全问题

### DISCUSSION THREAD

**Discussion by fmkli** [walinejs/waline#1172] [Q&A] [answered]
由于waline的评论是支持<img>标签的，我的网站使用waline时，我使用了某个XXS注入平台的图片注入测试，成功的抓取到了登录用户cookie及ip，希望waline对此进行改进

**Comment by fmkli** [comment]
img标签，被解析掉了啊这

**Comment by lizheming** [comment, marked as answer]
1. 我们的 HTML 应该是不支持直接展示的，会被转义成字符串出现
2. 如果是图片注入的话我们是支持插入图片的。但我理解你反馈的这个问题应该只要支持这个功能都会有这个特性，插入一个域名下的图片，会携带这个域名下的 COOKIE（当然 SameSite 的话会不携带）以及用户 IP 这都是 HTTP 标准里定义的，属于正常的
3. Waline 并不是使用 COOKIE 标记登录用户的

**Comment by fmkli** [comment]
明白，感谢解答
